### PR TITLE
Added analysis routines for time slicing, and played with Hough trans…

### PR DESCRIPTION
**Pull request by jrmlhermitte**
added some analysis code to help with time slicing of germ. incomplete.

### CODE REVIEW

File: config/99-germ-analysis.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+# chunk into 100 times
+N_TIMES = 100
+
+N_ENERGY = 4096
+MAX_ENERGY = 70
+
+from skbeam.core.accumulators.histogram import Histogram
+
+hh = Histogram((MAX_CHIP+1, 0, MAX_CHIP+1),
```
Comment (jrmlhermitte) [inline config/99-germ-analysis.py:148]: Central piece is the use of scikit-beam `Histogram`

Comment (tacaswell) [reply config/99-germ-analysis.py:148]: Did you investigate doing this with pandas as well?

Comment (jrmlhermitte) [reply config/99-germ-analysis.py:148]: no I did not. the idea here is to eventually be able to fill results on the fly.
Would pandas do that? I only see `.hist` which creates a 1D histogram.

Comment (tacaswell) [reply config/99-germ-analysis.py:148]: you can do this with groupby (like how it is done in heatmap).  `toolz` has support for streaming groupby I think....

**Pull request closed without merging**